### ISSUE TITLE
Enqueue scripts and manually initialize DFV for media library

### ISSUE THREAD

**Issue by pglewis** [pods-framework/pods#4785]
Related: #4462, #4729

PR: #4795

**Comment by pglewis** [comment]
This should hopefully allow pick field views to work in the Media Library's grid mode, with the exception of autocomplete and list view (due to some technical complications).

**Comment by pglewis** [comment]
@andreas2249, @GitHubGreg: I should have a working commit here soon, just waiting on a couple other merges ahead of it.  

From my local tests, it should be back to working at least as well as the 2.6.x branch was.  Custom pre-defined lists and relationships to other content types worked for me with radio buttons or dropdown... should work for anything except autocomplete or list view.

**Comment by pglewis** [comment]
PR should be good to go, just pending tests to merge into 2.x

**Comment by pglewis** [comment]
And merged, should be fixed in the 2.x branch... at least for the things that worked in 2.6.x.
